src/cli/commands/public-cmds/show-cmd.ts

```diff
@@ -126,6 +126,11 @@ export default class Show extends Command {
           // @ts-ignore AUTO-ADDED-AFTER-MIGRATION-PLEASE-FIX!
           componentObj.tester.files = makeEnvFilesReadable(comp.tester);
         }
+
+        if (comp.componentMap && comp.componentMap.rootDir) {
+          componentObj.rootDir = comp.componentMap.rootDir;
```

Comment:
@guysaar223 you should use getComponentDir
or maybe getTrackDir
(depends on what exactly you want from that dir, one also takes into account a wrap dir if exist)

Reply:
@GiladShoham 
fixed can you please check it.